builtin/logical/pki/path_roles.go

```diff
@@ -286,6 +286,11 @@ for "generate_lease".`,
 				Type:        framework.TypeBool,
 				Description: `Mark Basic Constraints valid when issuing non-CA certificates.`,
 			},
+			"not_before": &framework.FieldSchema{
+				Type:        framework.TypeInt,
+				Default:     30,
```

Comment:
Would you mind removing the default from both the backend and frontend definitions?  If this value is the default zero value, it should just ignore this field.